File: BusinessTripApplicationExtensions/BusinessTripApplicationServerExtension/Feature1/Services/IBusinessTripAppService.cs
using System;
using BusinessTripApplicationServerExtension.Feature1.Models;
using DocsVision.Platform.WebClient;

namespace BusinessTripApplicationServerExtension.Feature1.Services
{

    public interface IBusinessTripAppService
    {
        BusinessTripAppNameModel GetComPersInfo(SessionContext sessionContext, BusinessTripAppNameRequestModel model);
        BusinessTripAppNameModel GetCityInfo(SessionContext sessionContext, BusinessTripAppNameRequestModel model);
        void InitBusinessTrip(SessionContext sessionContext, Guid cardId);
    }
}

File: BusinessTripApplicationExtensions/BusinessTripApplicationWebExtension/src/Services/Interfaces/IBusinessTripAppService.ts
import { serviceName } from "@docsvision/webclient/System/ServiceUtils";
import { IBusinessTripAppNameModel } from "../../Model/IBusinessTripAppNameModel";
import { IBusinessTripAppNameRequestModel } from "../../Model/IBusinessTripAppNameRequestModel";

export interface IBusinessTripAppService {
    
    GetComPersInfo(model: IBusinessTripAppNameRequestModel): Promise<IBusinessTripAppNameModel>;
    GetCityInfo(model: IBusinessTripAppNameRequestModel): Promise<IBusinessTripAppNameModel>;
}

export type $BusinessTripAppService = { activityPlanService: IBusinessTripAppService };
export const $BuisnessTripAppService = serviceName<$BusinessTripAppService, IBusinessTripAppService>(x => x.activityPlanService);